dao.py

```diff
@@ -23,13 +23,13 @@ def _get_by_params(self, collection, selector):
     def _save_one(self, collection, document):
         res = self.__db[collection].insert_one(document)
         if res.acknowledged:
-            return res.inserted_id
+            return str(res.inserted_id)
         return None
 
     def _save_many(self, collection, documents_list):
         res = self.__db[collection].insert_many(documents_list)
         if res.acknowledged:
```

Comment:
just return the acknowledgement .. `return res.acknowledged` is totally fine here .. same with `_save_one`